extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -336,6 +336,25 @@ describes.realWin('amp-story', {
     expect(nextStub).calledOnce;
     expect(prevStub).calledOnce;
   });
+
+  it('runs vsync on resize', () => {
+    element.offsetWidth = 10;
+    element.offsetHeight = 9;
+    const isDesktopStub = sandbox.stub(story, 'isDesktop_').returns(false);
+    const vSyncStub = sandbox.spy(story.vsync_, 'run');
+    story.onResize();
+    expect(isDesktopStub).to.be.calledOnce;
+    expect(vSyncStub).to.be.calledOnce;
```

Comment:
This only checks for `vsync.run` to be called. Ideally, we'd stub `vsync.run` with a synchronous function that runs `measure` and `mutate` in sequence, and then assert the DOM state with our expectations.

Reply:
u sure? cuz it wasn't happening like this..
will check

Reply:
[Here's an example.](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/0.1/test/test-amp-form.js#L257) You'll see that it runs measure/mutate tasks synchronously and then asserts state expectations (e.g. `expect(form.hasAttribute('amp-novalidate')).to.be.true`, etc.).

Reply:
Done thanks 😄 .

Wow tests can get painful at times 😆